### PR TITLE
Add a CONTRIBUTORS file

### PR DESCRIPTION
**Pull request by alistair23**
The list of people was generated with:
    git log | grep Signed | sort | uniq

Also update the LICENSE file to point to the new file.

Signed-off-by: Alistair Francis <alistair.francis@wdc.com>

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2017 Khem Raj
+Copyright (c) 2020 Alistair Francis <alistair.francis@wdc.com>
```
Comment (kraj) [inline LICENSE:4]: I wonder if its better to remove the Copyright section from here completely and add a CONTRIBUTORS file listing names of all contributors

File: CONTRIBUTORS [hunk 2]
```diff
@@ -0,0 +1,17 @@
+List of meta-riscv contributors, sorted by Alphabetical order
+
+ * Alistair Francis <alistair.francis@wdc.com>
+ * Chin Huat Ang <chinhuat@gmail.com>
+ * Jeremy Puhlman <jpuhlman@mvista.com>
+ * Khem Raj <raj.khem@gmail.com>
+ * Mirza Krak <mirza.krak@endian.se>
+ * Moritz Fischer <moritz.fischer@ettus.com>
+ * Palmer Dabbelt <palmer@sifive.com>
+ * pino-kim <sungwon.pino@gmail.com>
```
Comment (pino-kim) [inline CONTRIBUTORS:10]: First, thanks to add my e-mail to CONTRIBUTE file. pino-kim and sungwon.pino is same peopele. would you please remove line number 10.?

Comment (alistair23) [reply CONTRIBUTORS]: Done.

Comment (pino-kim) [reply CONTRIBUTORS]: @alistair23  Thanks.

**Pull request merged**